bc_obps/reporting/json_schemas/2024/pulp_and_paper_production/pulp_and_paper_production.json

```diff
@@ -0,0 +1,53 @@
+{
+  "type": "object",
+  "title": "Pulping and chemical recovery",
+  "properties": {
+    "emissions": {
+      "type": "array",
+      "title": "Emissions",
+      "items": {
+        "title": "Pulping and chemical recovery",
+        "type": "object",
+        "properties": {
+          "carbonateName": {
+            "type": "string",
+            "title": "Carbonate Name"
+          },
+          "annualAmount": {
+            "type": "number",
+            "title": "Annual Amount (t)"
+          },
+          "purityOfCarbonate": {
+            "type": "number",
+            "title": "Purity Of Carbonate (Weight Fraction)"
+          },
```

Comment:
Looking at the excel sheet, these first 3 fields should be adjacent to the emissions array rather than be items within it. Moving these out of the array will require some changes to your uiSchema on the frontend. Let me know if this doesn't make sense & I'm happy to chat!

Reply:
Pulled these out into the activity with 46fcf40.